mico-core/src/main/java/io/github/ust/mico/core/broker/MicoApplicationBroker.java

```diff
@@ -327,14 +330,20 @@ public MicoServiceDeploymentInfo updateMicoServiceDeploymentInformation(String a
 
         int oldReplicas = storedServiceDeploymentInfo.getReplicas();
 
-        // Update existing service deployment information, reuse existing topics if and save it in the database.
         MicoServiceDeploymentInfo sdiWithAppliedValues = storedServiceDeploymentInfo.applyValuesFrom(serviceDeploymentInfoDTO);
-        MicoServiceDeploymentInfo sdiWithReusedTopics = reuseExistingTopics(sdiWithAppliedValues);
-        MicoServiceDeploymentInfo updatedServiceDeploymentInfo = serviceDeploymentInfoRepository.save(sdiWithReusedTopics);
-
-        // As a workaround it's necessary to save the same entity twice,
-        // because otherwise it sometimes happens that the relationship entity `MicoTopicRole` has no properties.
-        serviceDeploymentInfoRepository.save(updatedServiceDeploymentInfo);
+        // At first save the service deployment information with the new values without topics.
```

Comment:
Maybe add a FIXME to the comment and a link to the relevant neo4j issue (if there exist such an issue)